acceptance-tests/SUBMODULES.json

```diff
@@ -1,8 +1,8 @@
 [
   {
     "name": "roslyn", 
-    "url": "git://github.com/mono/roslyn.git",
-    "rev": "380eec412868464a132cdcd3a45ef457fab6e060",
+    "url": "git@github.com:mono/roslyn.git",
```

Comment:
please revert this back to git:// so it can be cloned anonymously.

Reply:
Ohhh, so that's the difference. Thanks, will do